src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -141,6 +150,21 @@ export class SkyListViewGridComponent
   }
 
   public ngAfterContentInit() {
+
+    // Watch for selection changes and update _multiselectSelectedIds for local comparison.
+    this.state.map(s => s.selected.item)
+      .takeUntil(this.ngUnsubscribe)
+      .distinctUntilChanged(this.selectedMapEqual)
+      .subscribe((items: ListSelectedModel) => {
+        let selectedIds: string[] = [];
+        items.selectedIdMap.forEach((isSelected, id) => {
```

Comment:
Personal preference here but I'd like to see a line break on each side of the `forEach` block as it is easier to read.

Reply:
yeah i think i can agree with that! updated.